### PR TITLE
Update to new clima parameters

### PR DESCRIPTION
**Pull request by trontrytel**
This PR updates us to the new way of handling CLIMAParameters. In the new approach we get the parameters by creating a toml file with all the needed parameters. We later put them into parameters, microphysics_parameters and thermodynamics_parameters "boxes" our relevant modules. 

We need CLIMAParameters to get the default values in tests. 

I also moved `t1` and `w1` from a separate struct into parameters.

@edejong-caltech and @sajjadazimi - lmk if it makes sense. To me it's a little bit more convoluted than before, hence more boiler plate. But I think we should nevertheless move on and strive to be compatible with the newest versions of clima packages. Also, this will deal with all the PRs that compat was trying to have

### CODE REVIEW

File: test/create_parameters.jl [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        println(io, "[prescribed_flow_t1]")
+        println(io, "alias = \"t1\"")
+        println(io, "value = " * string(t1))
+        println(io, "type = \"float\"")
+    end
+    toml_dict = CP.create_toml_dict(FT; override_file, dict_type="alias")
+    isfile(override_file) && rm(override_file; force=true)
+
+    aliases = string.(fieldnames(TD.Parameters.ThermodynamicsParameters))
+    aliases = (aliases..., "cp_d")
```
Comment (sajjadazimi) [inline test/create_parameters.jl:63]: cp_d is derived from other parameters and is not available in any toml file (neither default nor here); I don't understand why you extend aliases here by cp_d. I think get_parameter_values! doesn't return any value for cp_d. Maybe you can just remove this line.

File: src/Kinematic1D.jl [hunk 2]
```diff
@@ -23,6 +21,10 @@ const CMNe = CM.MicrophysicsNonEq
 const CM0 = CM.Microphysics0M
 const CM1 = CM.Microphysics1M
 
+include("Parameters.jl")
+import .Parameters as KP
+const APS = KP.AbstractKinematicParameters
```
Comment (sajjadazimi) [inline src/Kinematic1D.jl:26]: APS is not used anywhere. Consider removing this line

**Review by sajjadazimi — APPROVED** [review]
Thanks for making these changes. In general I think the idea is solid. There are a couple of minor comments about unused variables:
- Parameters.jl line 14
- test/create_parametrs.jl line 63
change if you think they make sense otherwise merge (I approve it anyways)

File: src/Parameters.jl [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+w1(ps::AKP) = ps.w1
+t1(ps::AKP) = ps.t1
+
+# Forward parameters to Thermodynamics
+const TDPS = TD.Parameters.ThermodynamicsParameters
+for var in fieldnames(TDPS)
+    @eval $var(ps::AKP) = TD.Parameters.$var(thermodynamics_params(ps))
+end
+# Forward parameters to CloudMicrophysics
+ρ_cloud_liq(ps::AKP) = CM.Parameters.ρ_cloud_liq(microphysics_params(ps))
```
Comment (sajjadazimi) [inline src/Parameters.jl:35]: This function is not used. Unlike thermo parameters, microphysics parameters are not used individually in the KiD module. Maybe we don't need to forward parameters to CloudMicrophysics.

File: test/create_parameters.jl (hunk 1 above)
Comment (trontrytel) [reply test/create_parameters.jl:63]: Right, good catch. At first I tried to provide a value for `cp_d`, but then realised its derived. So in the end I overwrite the `kappa_d` value, to get the `cp_d` same as in PySDM. 

I forgot to delete and will do so now.

File: src/Kinematic1D.jl (hunk 2 above)
Comment (trontrytel) [reply src/Kinematic1D.jl:26]: done

**Pull request merged**